### ISSUE TITLE
aria-label attribute disallowed on iframe

### ISSUE THREAD

**Issue by jmedwards-indeed** [dequelabs/axe-core#2945]
Product: axe-core

Expectation: `aria-label` can be used to supply an accessible name for an `iframe` (with no explicit role and no `title` attribute) 
Actual: Issue flagged as Critical:

> **Elements must only use allowed ARIA attributes**
> *Issue Description*
> Ensures ARIA attributes are allowed for an element's role

> *Element location*
> `iframe`
> *Element source*
> `<iframe class="cmp-ResponsiveVimeoVideo-iframe" src="https://player.vimeo.com/video/336812686?dnt=true" frameborder="0" allowfullscreen="" aria-label="Video player" data-ready="true"></iframe>`
> *To solve this issue, you need to...*
> *Fix the following:*
> ARIA attribute cannot be used: aria-label

Motivation: Use of `aria-label` on `iframe` appears to be acceptable under [applicable standards](https://www.w3.org/TR/wai-aria-1.1/#aria-label), and some sources [suggest](https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/) or [recommend](https://accessibilityinsights.io/info-examples/web/frame-title/) this attribute as an alternative to `title` (particularly in cases where a tooltip is undesirable).

<pre><code>
axe-core version: 4.2.0
(tested with) axe DevTools version: 4.9.3
</code></pre>

**Comment by straker** [comment]
Thanks for the issue. I believe this is related to #2933 and #2926 and should be fixed by #2929 and #2935

**Comment by WilcoFiers** [comment]
Firstly, you should probably use `title` attributes for iframes. Unfortunately on Safari, the `aria-label` attribute is ignored.

Steve is right though, we introduced a bug in axe-core 4.2.0. We released a fix for it yesterday with 4.2.1 which we're rolling out everywhere in the next week or so.
